src/js/yeldaWebchatInjector.js

```diff
@@ -104,6 +110,16 @@ class YeldaChat {
     this.webChatContainer.appendChild(this.assistantBubbleText)
   }
 
+  /**
+   * Callback function for bubble text close button
+   * @param {Event} event
+   */
+  closeBubbleText(event) {
+    event.preventDefault()
+    event.stopPropagation()
```

Comment:
isn't it easier to have a `href="#"` in the component it self ? 
it would be more w3c respectfull